images/core.proto

```diff
@@ -81,6 +81,10 @@ message thread_sas_entry {
 	required uint32			ss_flags	= 3;
 }
 
+message thread_allowedcpus_entry {
+	repeated uint64			cpumask		= 1;
```

Comment:
For historical reasons, repeated fields of scalar numeric types aren't encoded as efficiently as they could be. New code should use the special option `[packed=true]` to get a more efficient encoding. For example:
```
repeated int32 samples = 4 [packed=true];
```
https://developers.google.com/protocol-buffers/docs/proto#optional